README.md

```diff
@@ -171,630 +171,476 @@ curl https://raw.githubusercontent.com/grafana/helm-charts/main/charts/loki-stac
-EOF
+  hostname: echo.mandrakee.xyz
+  acmeProvider:
+    email: echo@gmail.com
+  tlsSecret:
+    name: tls2-cert
+  requestPolicy:
+    insecure:
+       action: Redirect
+       additionalPort: 8080
 
 ```
 
-Now I see the actual client IP at the pod.
+It takes ~30 seconds to get the signed certificate for the hosts. At this point, we have Ambassador installed, and hosts configured. BUT, we still do not have the networking (eg. DNS and Load balancer) configured to route the traffic to the cluster. 
 
-```
-~ curl -i http://$LB_ENDPOINT/echo/
-HTTP/1.1 200 OK
-content-type: text/plain
-date: Mon, 28 Jun 2021 05:48:48 GMT
-content-length: 359
-x-envoy-upstream-service-time: 5
-server: envoy
-
-Request served by echo-deployment-869b7bf9c7-r5mmd
-
-HTTP/1.1 GET /
-
-Host: 164.90.247.161
-Accept: */*
-X-Envoy-External-Address: 73.162.138.xxx
-X-Request-Id: 04e0e2a8-8b3b-474d-bc01-7a68d0bb01ed
-User-Agent: curl/7.64.1
-X-Forwarded-For: 73.162.138.xxx
-X-Forwarded-Proto: http
-X-Envoy-Expected-Rq-Timeout-Ms: 3000
-X-Envoy-Original-Path: /echo/
-Content-Length: 0
 
-```
+### Configure your domain mapping to point the domains to the cluster LB
 
-#### Getting traffic in using DNS name
-This is fairly straightforward. We will just add an A record for every host into the DNS. In my case, I added 2 hosts (echo and test) pointing to the same LB.
+Here  hosting 2 hosts (echo.mandrakee.xyz, quote.mandrakee.xyz) on the same cluster.
 
 ```
-~ doctl compute domain records list kubenuggets.dev
-ID           Type    Name    Data                    Priority    Port    TTL     Weight
-158200732    SOA     @       1800                    0           0       1800    0
-158200733    NS      @       ns1.digitalocean.com    0           0       1800    0
-158200734    NS      @       ns2.digitalocean.com    0           0       1800    0
-158200735    NS      @       ns3.digitalocean.com    0           0       1800    0
-158200782    A       echo    164.90.247.161          0           0       180     0
-158201299    A       test    164.90.247.161          0           0       180     0
+~ doctl compute domain records list mandrakee.xyz  
+ID           Type    Name    Data                    Priority    Port    TTL     Weight  
+158200732    SOA     @       1800                    0           0       1800    0  
+158200733    NS      @       ns1.digitalocean.com    0           0       1800    0  
+158200734    NS      @       ns2.digitalocean.com    0           0       1800    0  
+158200735    NS      @       ns3.digitalocean.com    0           0       1800    0  
+158200782    A       echo    143.244.208.164         0           0       180     0  
+158201299    A       quote   143.244.208.164         0           0       180     0  
+~   
+~ kgsvcn ambassador   
+NAME               TYPE           CLUSTER-IP       EXTERNAL-IP       PORT(S)                      AGE  
+ambassador         LoadBalancer   10.245.215.249   143.244.208.164   80:32723/TCP,443:31416/TCP   23h  
```

Comment:
Hide the external IP in the output. Just mark it as xxx.xxx.

Reply:
done!